docs/README.md

```diff
@@ -81,10 +81,11 @@ for more information about installing and using the source code.
 
 A collection of Angular components for generic use.
 
-| Name                                          | Description                  |
-|-----------------------------------------------|------------------------------|
-| [Avatar](core/components/avatar.component.md) | Displays user avatars.       |
-| [Button](core/components/button.component.md) | A standard button component. |
+| Name                                                | Description                                     |
+|-----------------------------------------------------|-------------------------------------------------|
+| [Avatar](core/components/avatar.component.md)       | Displays user avatars.                          |
+| [Button](core/components/button.component.md)       | A standard button component.                    |
+| [UI Header](core/components/ui-header.component.md) | A simple reusable application header component. |
```

Comment:
can we make it just `Header`?

Reply:
of course, should I also rename the component selector and file names?

Reply:
yeah, just `HeaderComponent` (`adf-header`) should work

Reply:
done, ty